scripts/create_metadata

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+#Check if DEV_TAGS is unset, if not
+if [ -z "${DEV_TAGS-}" ]; then
```

Comment:
I think this should be like this, with a colon:

```suggestion
if [ -z "${DEV_TAGS:-}" ]; then
```

Reply:
thanks! I updated it with the suggested change.